dotnet/src/Agents/OpenAI/Internal/AssistantThreadActions.cs

```diff
@@ -611,7 +619,15 @@ private static StreamingChatMessageContent GenerateStreamingMessageContent(strin
         // Process annotations
         else if (update.TextAnnotation != null)
         {
-            content.Items.Add(GenerateStreamingAnnotationContent(update.TextAnnotation));
+            StreamingAnnotationContent? annotationItem = GenerateStreamingAnnotationContent(update.TextAnnotation);
```

Comment:
Same as above, using the same signature expecting the `StreamingAnnotationContent` instead
```suggestion
            StreamingAnnotationContent? annotationItem = GenerateStreamingAnnotationContent(update.TextAnnotation);
```

Reply:
Same dynamics as previous comment.

> NOTE: I'm not seeing any delta in your suggestion.